release/ml_user_tests/horovod/horovod_user_test.py

```diff
@@ -22,6 +22,7 @@
         num_workers=6,
         use_gpu=True,
         placement_group_timeout_s=2000,
+        timeout_s=300,
```

Comment:
I also verified that this works with 120 seconds, if we want to be more cautious about increasing it too much from the default 30 seconds.